Makefile

```diff
@@ -400,16 +364,13 @@ changelog:
 .PHONY: install-tools
 install-tools:
 	go install github.com/wadey/gocovmerge
-	go install golang.org/x/lint/golint
 	go install github.com/mjibson/esc
-	go install github.com/securego/gosec/cmd/gosec
-	go install honnef.co/go/tools/cmd/staticcheck
 
 .PHONY: install-ci
 install-ci: install-tools
 
 .PHONY: test-ci
-test-ci: build-examples lint cover
+test-ci: build-examples cover
```

Comment:
I would consider killing this target and invoking directly in GA

Reply:
I like a minimal CI configuration. I want to be able to easily replicate locally what CI is doing - e.g. run the same make target.

Reply:
Yup, I agree that it's a better developer experience if the CI scripts can be traced down to the make target then "replayed" locally, which a GA step doesn't afford.

As such, I've removed the GA step to run golangci-lint and kept the original setup where the `test-ci` target will run the `lint` target. Fewer changes, and cleaner.